debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/connection/PostgresReplicationConnection.java

```diff
@@ -321,15 +321,15 @@ public void initConnection() throws SQLException, InterruptedException {
 
     @Override
     public Optional<SlotCreationResult> createReplicationSlot() throws SQLException {
-        // note that some of these options are only supported in pg94+, additionally
-        // the options are not yet exported by the jdbc api wrapper, therefore, we just do this ourselves
-        // but eventually this should be moved back to the jdbc API
+        // note that some of these options are only supported in Postgres 9.4+, additionally
+        // the options are not yet exported by the jdbc api wrapper, therefore, we just do
+        // this ourselves but eventually this should be moved back to the jdbc API
         // see https://github.com/pgjdbc/pgjdbc/issues/1305
 
         LOGGER.debug("Creating new replication slot '{}' for plugin '{}'", slotName, plugin);
         String tempPart = "";
-        // Exported snapshots are supported in PostgreSQL 9.4+
-        Boolean canExportSnapshot = pgConnection().haveMinimumServerVersion(ServerVersion.v9_4);
+        // Exported snapshots are supported in Postgres 9.4+
```

Comment:
The database is really named PostgreSQL :-)

Reply:
I know. A PostgreSQL committer was my professor at university. Still, we tend to use Postgres in most places.